packages/editor/src/components/browser/TypeBrowser.tsx

```diff
@@ -80,19 +98,23 @@ const TypeBrowser = ({ value, onChange, onDoubleClick, initSearchFilter }: TypeB
       const ownTypesWithoutDataClasses = ownTypes.filter(
         ownType => !mappedDataClasses.find(dataClass => dataClass.fullQualifiedName === ownType.fullQualifiedName)
       );
-      const mappedOwnTypes: TypeBrowserObject[] = ownTypesWithoutDataClasses.map<TypeBrowserObject>(ownType => ({
-        icon: IvyIcons.DataClass,
-        ...ownType
-      }));
-
-      const sortedIvyTypes = mappedIvyTypes.sort(typeComparator);
+      const sortedIvyTypes = mappedIvyTypes.sort((a, b) => typeComparator(a, b, true));
       const sortedMappedDataClasses = mappedDataClasses.sort(typeComparator);
-      const sortedMappedOwnTypes = mappedOwnTypes.sort(typeComparator);
+      if (location.includes('code')) {
+        const mappedOwnTypes: TypeBrowserObject[] = ownTypesWithoutDataClasses.map<TypeBrowserObject>(ownType => ({
+          icon: IvyIcons.DataClass,
+          ...ownType
+        }));
+        const sortedMappedOwnTypes = mappedOwnTypes.sort(typeComparator);
 
-      const sortedTypes: TypeBrowserObject[] = sortedMappedOwnTypes.concat(sortedMappedDataClasses).concat(sortedIvyTypes);
-      setTypes(sortedTypes);
+        const sortedTypes: TypeBrowserObject[] = sortedMappedOwnTypes.concat(sortedMappedDataClasses).concat(sortedIvyTypes);
+        setTypes(sortedTypes);

```

Comment:
why save it first in a variable?
`setTypes(sortedMappedOwnTypes.concat(sortedMappedDataClasses).concat(sortedIvyTypes));`